maze_transformer/evaluation/plot_loss.py

```diff
@@ -52,6 +52,7 @@ def plot_loss(
     total_sequences, loss = zip(*data_raw)
 
     # compute a rolling average
+    # this messy part here is just for handling command line arguments
```

Comment:
Maybe worth extracting a method  called `parse_window_sizes` or something like that, then we wouldn't need this comment

Reply:
good point!